src/database-postgresql.h

```diff
@@ -0,0 +1,73 @@
+/*
+Minetest
+Copyright (C) 2014 jjb, James Dornan <james@catch22.com>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#include "database.h"
+#include "settings.h"
+#include <string>
+
+#include <postgresql/libpq-fe.h>
+#include <postgresql/libpq/libpq-fs.h>
+
+class ServerMap;
+
+class Database_PostgreSQL : public Database
+{
+public:
+	Database_PostgreSQL(const std::string &savedir);
+
+	virtual void beginSave();
+	virtual void endSave();
+
+	virtual bool saveBlock(const v3s16 &pos, const std::string &data);
+	virtual std::string loadBlock(const v3s16 &pos);
+	virtual void listAllLoadableBlocks(std::vector<v3s16> &dst);
+	virtual bool deleteBlock(const v3s16 &pos);
+	virtual bool updateBlock(const v3s16 &pos, const std::string &data);
+	virtual bool initialized(void);
+
+	~Database_PostgreSQL();
+private:
+	ServerMap *srvmap;
+	PGconn *m_conn;
+	PGresult *m_res;
+	Settings m_conf;
+	std::string m_savedir;
+	std::string m_dbname;
+	std::string m_user;
+	std::string m_password;
+	std::string m_host;
+	std::string m_conninfo;
+	std::string m_port;
+	int *row;
```

Comment:
No, members are prefixed with m_. Even @kwolekr now agrees with this. Perhaps if there was such a tendency in the MT source code to have super large functions then omitting the prefix might make sense, but because so many functions don't even fit on one screen it's PITA to see whether something is in scope or not. The style guidelines need updating.